storage/bucket_test.go

```diff
@@ -252,6 +253,7 @@ func TestBucketAttrsToRawBucket(t *testing.T) {
 				},
 			},
 		},
+		IpFilter: &raw.BucketIpFilter{Mode: "disabled"},
```

Comment:
Maybe this should include PublicNetworkSource/VPCNetworkSource fields to test the nested converters for these?